mlflow/tensorflow/callback.py

```diff
@@ -0,0 +1,87 @@
+from tensorflow import keras
+
+from mlflow import log_param, log_text
+from mlflow.utils.autologging_utils import BatchMetricsLogger
+
+
+class MLflowLoggingCallback(keras.callbacks.Callback):
+    """Callback for logging Tensorflow training metrics to MLflow.
+
+    This callback logs training metrics every epoch or every n steps (defined by the user) to
+    MLflow.
+
+    Args:
+        run: an 'mlflow.entities.run.Run' instance, the MLflow run.
+        log_every_epoch: bool, If True, log metrics every epoch. If False, log metrics every n
+            steps.
+        log_every_n_steps: int, log metrics every n steps. If None, log metrics every epoch.
+
+    Examples:
+        ```python
+        from tensorflow import keras
+        import mlflow
```

Comment:
Does this code block render fine on the documentation?

Reply:
there should be a way, but I am done to be consistent with other modules. changed.